scripts/kibana/validate-ts-interfaces-against-apm-server-sample-docs/scripts/clone-kibana.sh

```diff
@@ -5,5 +5,5 @@ echo "Cloning Kibana: $OWNER:$BRANCH"
 
 cd ./tmp
 git clone --depth 1 -b $BRANCH https://github.com/$OWNER/kibana.git
-mv ./kibana/x-pack/plugins/apm/typings/es_schemas ./apm-ui-interfaces
+mv ./kibana/x-pack/legacy/plugins/apm/typings/es_schemas ./apm-ui-interfaces
```

Comment:
we have to maintain the current behavior and support the new one

```suggestion
mv ./kibana/x-pack/plugins/apm/typings/es_schemas ./apm-ui-interfaces \
  || mv ./kibana/x-pack/legacy/plugins/apm/typings/es_schemas ./apm-ui-interfaces

```

Reply:
@kuisathaverat 	We're all expected to update our branches ASAP with the new folder paths. Usually this job runs only on `master`, which already has the new paths. Do you think it's worth it for the rare occasion where somebody wants to run this script on a branch that has old paths?

Reply:
I've double checked it, we do not run this test with any other versions than the master, so it is fine to replace the command as you suggested.